pkg/scheduler/schedule_one.go

```diff
@@ -373,7 +377,7 @@ func (sched *Scheduler) schedulePod(ctx context.Context, fwk framework.Framework
 		return result, err
 	}
 
-	host, err := selectHost(priorityList)
```

Comment:
As described in the PR description, the 2nd return value will be used later in another PR.